python/__init__.py.in

```diff
@@ -21,3 +23,61 @@ if sys.version_info >= (3, 8) and sys.platform == "win32":
 from .pyf3d import *
 
 __version__ = "@F3D_VERSION@"
+
+
+################################################################################
+# monkey patch `options.update`
```

Comment:
what does this do ? why do we need it ?

Reply:
allows to update the options by passing a mapping (most likely a `dict`) or key-value pairs:
```py
options.update({
    "interactor.axis": True,
    "model.material.roughness": 0.8,
    #...
})
# and
options.update([
    ("interactor.axis", True),
    ("model.material.roughness", 0.8),
    #...
])
```
which is quite convenient and python-ish, but it was a pain to implement purely on the `pybind` side.
So we write the method in python and add it to the generated class when we load the module.

Reply:
and I think it's fine :)  
I see no benefit doing it in pybind

Reply:
great, put some comment so that other devs can understand what is going on :)